pinc/image_check.inc

```diff
@@ -12,3 +13,14 @@ function get_image_size_error($image_size)
     }
     return $result;
 }
+
+function get_image_size_small_error($image_dimensions)
+{
+    global $page_image_minimum_size;
+    $result = null;
+
+    if ($image_dimensions[0] < $page_image_minimum_size || $image_dimensions[1] < $page_image_minimum_size) {
+        $result = _("The image may be too small: the shortest dimension of the images is less than $page_image_minimum_size pixels.");
```

Comment:
To properly translate this string, it needs to use `sprintf()` for the variable.

Ditto the other instance in `project_quick_check.inc`.